src/mappings/entities/mapClaim.ts

```diff
@@ -89,75 +106,22 @@ export async function mapClaim(
     }
   }
 
+  if (eventId === EventIdEnum.ClaimRevoked) {
+    const id = getId(target, claimType, scope, jurisdiction, cddId);
+    const claim = await Claim.get(id);
+    if (!claim) {
+      throw new Error(`Claim with id ${id} was not found`);
+    }
+    claim.revokeDate = issuanceDate;
+    await claim.save();
+  }
+
```

Comment:
What do you think about refactoring this in the following shape:
- A `switch case` to make it clearer which `eventIds` it is covering and which it isn't.
- A `function` for each one so the code for the mapping is very easy to read. 

Thoughts @prashantasdeveloper @polymath-eric? If you are OK with this, it isn't necessary to apply it now, we could add a task to do it later.

Reply:
Refactored using multiple ifs